### PR TITLE
Export img

### PR DESCRIPTION
**Pull request by kenneth-miura**
# Description

Allows you to export to jpg. If both fall and winter session in schedule, generates two jpgs

image

Fixes # (issue) 

## Type of change

- [x] New feature 

# How Has This Been Tested?
Manual tests

# Checklist:

- [x] I ran `./gradlew spotlessApply`
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I ran `./gradlew build` and the gradle build succeeded
- [x] I have not violated any design principles

# Other Comments:

Other comments that should be made clear.

### CODE REVIEW

**Comment by kenneth-miura** [comment]
I can add tests for the exporter, in terms of verifying it'll only create a winter schedule if the generated schedule has winter courses, and same for fall. Is that worth it?

**Review by roryhofland — APPROVED** [review]
I really like this, and I think Chris will too. I tried it with my classes this semester and it gave me my exact timetable.

File: src/main/java/controllers/CommandLineInterface.java [hunk 1]
```diff
@@ -471,6 +473,12 @@ public Schedule displayUserSchedules(List<Schedule> userSchedules) {
                     Exporter exporter = new CSVExporter();
                     exporter.outputSchedule(currSchedule, csvFileName);
                     break;
+                case 'J':
+                    System.out.println(
+                            "Please specify the name you'd like to save this Schedule under.");
```
Comment (lorena-b) [inline src/main/java/controllers/CommandLineInterface.java:478]: make this WHITE_BOLD

File: src/main/java/controllers/CommandLineInterface.java [hunk 2]
```diff
@@ -471,6 +473,12 @@ public Schedule displayUserSchedules(List<Schedule> userSchedules) {
...
+                    String jpgFileName = scanner.next();
+                    System.out.printf("Saving this schedule in .jpg format...");
```
Comment (lorena-b) [inline src/main/java/controllers/CommandLineInterface.java:480]: green

**Review by lorena-b — CHANGES_REQUESTED** [review]

**Comment by lorena-b** [comment]
I think this looks really good! Adding tests for the exporter would be good because it will get the coverage up

File: src/main/java/workers/ImageExporter.java [hunk 3]
```diff
@@ -0,0 +1,227 @@
...
+        courseIDs.add("TST104Y");
+        courseIDs.add("CSC236F");
+        courseIDs.add("CSC258S");
+        List<Course> courses = Controller.courseInstantiator(courseIDs);
+
+        Schedule schedule = new Scheduler().createBasicSchedule(courses);
+        System.out.println(schedule);
+        new ImageExporter().outputSchedule(schedule, "custom name");
+
+    }
```
Comment (lorena-b) [inline src/main/java/workers/ImageExporter.java:226]: was this for demo purposes?

**Comment by kenneth-miura** [comment]
ok im going to add tests in a separate PR later, want to get this merged in for demo mon

**Pull request merged**